.github/workflows/releases.yml

```diff
@@ -16,10 +22,14 @@ jobs:
       uses: actions/setup-node@v3
       with:
         node-version: '18.x'
+    - name: Install qemu dependency
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y qemu-user-static
```

Comment:
As agreed in a separate comment, we shouldn't emulate arm so this isn't necessary.